CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+
+### The purpose of the DReps
+
+Nothing in this proposal limits the SPOs from becoming DReps.
+Why do we have DReps at all?
+SPOs are chosen purely for block production.
+Voters can choose to delegate their vote to DReps without needing to consider whether they are
+also a good block producer.
+
+### AVST (action voting stake threshold)
+
+The AVST exists to ensure that the votes of the DReps are meaningful.
+For example, if only 10 Lovelace were properly delegated to DReps, and 9 Lovelace "vote yes" to
+a governance action, we would doubt the legitimacy of the result since there are
+billions of ADA in circulation.
+A high enough AVST legitimizes the representation.
+
+### Ratification Requirements Table
+
+The requirements in the [ratification requirement table](#requirements) are explained here.
+Most of the governance actions have the same kind of requirements:
+the constitutional committee must reach quorum and the DReps must reach a sufficient number of
+'yes' votes, except in the situation where the AVST is too low, in which case the SPOs vote in
+place of the DReps.
+This includes these actions:
+* New Committee/quorum (normal state)
+* Update to the Constitution
+* Protocol parameter changes
+* Treasury withdrawal
+
+#### Motion of no-confidence
+
+A motion of no-confidence represents the lack of confidence by the Cardano community in the
+current Constitutional Committee, and hence the Constitutional Committee should have no
+say whatsoever in this governance action.
+In this situation, the SPOs and the DReps are left to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
+
+#### New Committee/quorum (state of no-confidence)
+
+Similar to the motion of no-confidence, electing a constitutional committee
+depends on both the SPOs and the DReps to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
+
+#### Hard-Fork initiation
+
+Regardless of any governance mechanism, SPO participation is needed for any hard fork since it is
+they who must upgrade their software.
+For this reason, we make their cooperation explicit in the hard fork initiation governance action,
+by always requiring their vote.
+The Constitutional Committee also votes, signaling the constitutionality of a hard fork.
+The DReps also vote, to represent the will of every stake holder, though the AVST is not required.
+Requiring the AVST in this situation could block a critical upgrade,
+though we may wish to reconsider this in the future.
+
+### New Metadata structures
```

Comment:
I'd like to see a discussion of the "use transaction metadata" alternative. Here's a draft:

```
##### Alternative: Use of transaction metadata

Instead of specific dedicated fields in the transaction format, we could instead use the existing transaction metadata field.

Governance-related metadata can be clearly identified by registering a CIP-10 metadata label.
Within that, the structure of the metadata can be determined by this CIP (exact format TBD), using an index to map the vote or proposal id to the corresponding metadata URL and hash.

This avoids the need to add additional fields to the transaction body, at the risk of making it easier for submitters to ignore.
However, since the required metadata can be empty (or point to a non-resolving URL), it is already easy for submitters to not provide metadata, and so it is unclear whether this makes the situation worse.
```

Reply:
I'll add it, thanks! I just remember another reason why I am not in favor of using the tx metadata: I want to store the governance URLs in the ledger state, as we do for stake pools, to make it easier for clients to use this information. We never use tx metadata on chain for anything (that is, IMHO, it's distinguishing mark).

Reply:
But this information is not actually needed for the ledger rules to function. I think it's a bit weird and a major departure from previous policy if the ledger state starts to become responsible for aggregating "extra" information that's nice for clients but not necessary for the ledger rules.

Reply:
that's a very solid point! there are a few exceptions to that "policy", though, such as stake pool metadata and the stake pool ranking data.